python/nammu/SOAPClient/HTTPRequest.py

```diff
@@ -0,0 +1,119 @@
+from email.mime.application import MIMEApplication
+from email.encoders import encode_7or8bit
+from email.mime.multipart import MIMEMultipart
+from email.mime.base import MIMEBase
+
+class HTTPRequest:
```

Comment:
In python 2 a class should always inherit from object if they don't inherit from anything else to get new style objects